geopandas/io/arrow.py

```diff
@@ -606,6 +643,19 @@ def _read_parquet(path, columns=None, storage_options=None, **kwargs):
         both ``pyarrow.fs`` and ``fsspec`` (e.g. "s3://") then the ``pyarrow.fs``
         filesystem is preferred. Provide the instantiated fsspec filesystem using
         the ``filesystem`` keyword if you wish to use its implementation.
+    bbox: tuple, optional
+        Bounding box to be used to filter selection from geoparquet data. This
+        is only usable if the data was saved with the bbox covering metadata.
+        Input is of the tuple format (xmin, ymin, xmax, ymax).
+    read_bbox_column: bool, default False
```

Comment:
This might be my misreading of the [spec](https://github.com/opengeospatial/geoparquet/blob/main/format-specs/geoparquet.md#covering), but I think as written this conflates two things. The `"covering"` field in the geoparquet metdata specifies that the only defined covering encoding at present, and additionally encodes the column name that the bbox covering information is stored in. By convention that column is called "bbox", but it doesn't necessarily have to be. 

I think we should call this option `read_covering_bbox` to be a bit clearer about what this is and also to match the convention on the write side (`write_covering_bbox`), and I would think  that the text below needs to be amended, as it's not whether `columns` contains `"bbox"` but rather whether columns contains the bbox column labeled in the `"covering"` metadata. i.e. we should be able to read a geoparquet with polygons to a column called `"bbox"`, and the `bbox` struct format in a column called "bbox_struct" provided the metadata is set correctly.

(based on your implementation of `_get_bbox_encoding_column_name`  I think you do already support this, but just the text doesn't align)

Happy to be corrected if I've got this wrong though.

Reply:
> This might be my misreading of the [spec](https://github.com/opengeospatial/geoparquet/blob/main/format-specs/geoparquet.md#covering), but I think as written this conflates two things.

@m-richards I am not directly understanding how this keyword is conflating two things (the explanation in the docstring or naming could be improved, but I mean regarding the behaviour). 

FWIW I made the suggestion below to rename it to `read_covering_column` to not make it bbox-specific (in the future there might be other types of coverings, for which we also don't want to read the column by default).

And it's a good point that we need to make this more robust for files that use a different field name

Reply:
Sorry, wasn't clear - I meant the documentation text for the field, not not the field itself!